projects/template.md

```diff
@@ -67,10 +73,40 @@ After the project is accepted, the Network Steward will pay XXX PKT
 
 ### Milestone 1
 
-> At this point, a user will be able to ...
+#### Success criteria
+> **Instructions**: This is where you write success criteria which will be used for verifying that
+milestone 1 was delivered correctly.
 
-> After a report is delivered on this milestone and it is approved by the NS, the NS will send XXX PKT.
+#### Payout
+After a report is delivered on this milestone and it is approved by the NS, the NS will send **TODO** PKT.
 
 ### Milestone 2
 
-...
\ No newline at end of file
+...
+
+## Disclosure
+I hereby submit this application in good faith and I attest that I have made no effort, nor do I
+intend to make effort, influence the Network Steward to accept this or any other project I have
+submitted.
+
+*Please check one or more:*
+
+1. [ ] A company is receiving the funds
+  1. [ ] Company has financial relationships with one or more reviewers: *specify whom*
+  * [ ] Company has no financial relationships with any reviewers: *specify whom*
```

Comment:
Odd to say that there is no relationship with any reviewers and then ask to specify?